bolt/about.py

```diff
@@ -2,6 +2,9 @@
 
 """
 project = u'bolt'
+description = """
```

Comment:
I added a description so we can read these value in `setup.py`; otherwise, versions get out of sync.